public/stylesheets/landing.css

```diff
@@ -0,0 +1,43 @@
+canvas {
```

Comment:
it actually is necessary to target like this in order to override bootstrap having done the same